lib/views/feedbacks_slack_view.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Aprb.Views.FeedbacksSlackView do
+
+  def render(event) do
+    %{
+      text: "#{event["subject"]["display"]} #{event["verb"]} #{event["properties"]["message"]} from #{event["properties"]["url"]}",
```

Comment:
yes, basically there are two things we need to do when connecting locally to staging RabbitMQ, we need to first change the queue name locally to something like `"ashkan_local_aprb_#{topic}_queue" [here](https://github.com/artsy/aprb/blob/master/lib/services/amq_event_service.ex#L24) so we don't consume staging events. This then will create a new queue in RabbitMQ staging, the issue is it won't delete the queue once we stop the connection, so we need to manually go to dashboard and delete the queue. I _think_ there are few options we can use to delete the queue if nothing is connected but that may not make sense for staging/production cases.